src/main/resources/application.properties

```diff
@@ -35,3 +35,6 @@ swagger.info.license=GNU LESSER GENERAL PUBLIC LICENSE,  Version 3, June 2007
 swagger.info.licenseUrl=https://www.gnu.org/licenses/lgpl-3.0.en.html
 swagger.info.termsOfServiceUrl=https://github.com/entando/entando-core/wiki
 swagger.info.basePackage=org.entando.plugins.pda.controller
+
+# Configs
+config.url=${CONFIG_URL:http://localhost:8083/config/@project.artifactId@}
```

Comment:
Is this the URL of the config service?

Reply:
`http://localhost:8083/config` is the url.
`http://localhost:8083/config/@project.artifactId@` resolves to `http://localhost:8083/config/entando-process-driven-plugin` which is the url for accessing this plugin's specific configurations.